nexus_constructor/transformation_view.py

```diff
@@ -62,6 +63,7 @@ def enable(self):
     def saveChanges(self):
         self.transformation.ui_value = self.transformation_frame.value_spinbox.value()
         # self.transformation.dataset = self.transformation_frame.magnitude_widget.value
```

Comment:
ah ok i now see why the code in ui_value is different. ignore my previous comment about incorrect logic, but could you create an issue to get transformation magnitudes and `ui_value`  working properly please?